docs/probes.md

```diff
@@ -45,7 +45,7 @@ metadata:
     kubevirt.io/vm: readiness-probe-vm
   name: readiness-probe
 spec:
-  running: true 
+  runStrategy: "Always" 
```

Comment:
Are the quotes necessary? Vision is cleaner without them.

Reply:
Yup, you're right :)
Done, thanks